### ISSUE TITLE
Printing Too Far to The Left

### ISSUE THREAD

**Issue by CKidd368** [ExLibrisGroup/SpineOMatic#58]
Hey, I'm printing spine labels and the issue I'm having is that 6/8 columns and rows print fine, but the last two columns and rows are shifted to the left too much. I adjusted the layout settings a few times but every adjustment I made would not fix the last two columns. Is there a way I can move the last two columns more to the right of the labels? There's 56 labels total, 8 rows and 7 columns. Thank you

**Comment by mgobat** [comment]
@CKidd368, are you doing "Batch Print"? Could you provide a screenshot of your "Pirnt Setup" tab so I can try to replicate the problem?  Thanks.

**Comment by CKidd368** [comment]
Yes, I'm printing multiple labels at once.  I attached photos of the settings I'm using as well as what the labels look like once printed. 

![20230522_135725](https://github.com/ExLibrisGroup/SpineOMatic/assets/134000569/03ea0625-1514-48d4-ad73-576b23781223)
![20230522_135741](https://github.com/ExLibrisGroup/SpineOMatic/assets/134000569/0f9a6099-fbfa-4631-9374-7cae59c5f113)
![20230522_140834](https://github.com/ExLibrisGroup/SpineOMatic/assets/134000569/41c633a8-1c1f-4ced-b421-c820658e0f73)
![20230522_141018](https://github.com/ExLibrisGroup/SpineOMatic/assets/134000569/18544bf3-a595-4570-8d90-6f1e7625364c)
